src/Auth0.ManagementApi/Clients/OrganizationsClient.cs

```diff
@@ -46,6 +46,25 @@ public Task<IPagedList<Organization>> GetAllAsync(PaginationInfo pagination)
             return Connection.GetAsync<IPagedList<Organization>>(BuildUri("organizations", queryStrings), DefaultHeaders, converters);
         }
 
+        /// <summary>
+        /// Retrieves a list of all organizations using checkpoint <paramref name="pagination"/>.
+        /// </summary>
+        /// <param name="pagination">Specifies <see cref="CheckpointPaginationInfo"/> to use in requesting checkpoint-paginated results.</param>
+        /// <returns>An <see cref="IPagedList{Organization}"/> containing the organizations.</returns>
+        public Task<IPagedList<Organization>> GetAllAsync(CheckpointPaginationInfo pagination)
+        {
+            if (pagination == null)
+                throw new ArgumentNullException(nameof(pagination));
+
+            var queryStrings = new Dictionary<string, string>
+            {
+                {"from", pagination.From?.ToString()},
+                {"take", pagination.Take.ToString()},
+            };
+
+            return Connection.GetAsync<IPagedList<Organization>>(BuildUri("organizations", queryStrings), DefaultHeaders, converters);
```

Comment:
Shouldnt this be `ICheckpointPagedList` rather than `IPagedList`? (same goes for the other lines)

Reply:
Fixed, thanks!